src/discord/commands/moderation/nickname.ts

```diff
@@ -0,0 +1,36 @@
+import {
+    Command, AccessLevel,
+    CommandError, Logger,
+    Guards
+} from "dd-botkit";
+
+export const NicknameCommand: Command = {
+    opts: {
+        name: "nickname",
+        access: AccessLevel.MODERATOR,
+        category: "Moderation",
+        guards: [
+            Guards.Argumented(
+                "nickname",
+                "Force changes the nickname of a user",
+                [{
+                    name: "user",
+                    type: "user",
```

Comment:
Use a member object directly to avoid having to get the member manually.

Reply:
So use `member` instead of `user`?